lib/rendering/renderables/renderable.cpp

```diff
@@ -2,8 +2,11 @@
 #include "brickengine/rendering/renderables/data/rect.hpp"
 #include "SDL2/SDL.h"
 
-Renderable::Renderable(int layer) :
-    layer(layer) {}
+Renderable::Renderable(int layer) : layer(layer) {
+    alpha = 255;
```

Comment:
You can use
`: layer(layer), alpha(255) {}`

Reply:
Ah cool, did not think of this.